README.md

```diff
@@ -60,14 +52,101 @@ Spinal cord MRI data:
 - T2star tra
 Brain MRI data:
 - DWI (A-P and P-A phase encoding)
-- MPM (multi-parameter mapping)
+- T1w
 
 ## naming convention
 
 sub-<site><pathology>XXX
 
 example:
-sub-zurichDCM001
+sub-montrealDCM001
```

Comment:
Is "DCM" from "dicom" or is it the name of a pathology?

Reply:
Degenerative Cervical Myelopathy

Reply:
no need to systematically add this info in the file name (we do it for the INSPIRED study because this is how the data were originally organized, and it is 'easy' to inspect)-- however, this info always needs to be present in the participants.tsv file, because we will use it to filter our multiple DBs for creating deep learning models

Reply:
Okay! Then, I'll upgrade the description as the following:

```
## naming convention

sub-<institution><pathology>XXX

example:
sub-montrealDCM001

Note: use <institution> and <pathology> only if relevant (e.g., multi-institution dataset or dataset with different pathologies). However, always keep <institution> and <pathology> in the participants.tsv file!!!
```